verif/core_rrv/regress/rv32i_level0_cfg_intrpt

```diff
@@ -1 +1,2 @@
-alive_illegal.c     -gRF_NUM_MSB=31
\ No newline at end of file
+alive_illegal.c     -gRF_NUM_MSB=31
+bubblesort_with_timer.c -gRF_NUM_MSB=31
```

Comment:
Why not add this to the level0 just to make sure it SW compiles and runs correctly

Reply:
I prepared to do that latter after we finish all he issues regardess timer_interrupt. But I understand, next time I will add it